src/box/lua/net_box.c

```diff
@@ -2307,10 +2359,15 @@ netbox_transport_do_id(struct netbox_transport *transport, struct lua_State *L)
 	ERROR_INJECT(ERRINJ_NETBOX_DISABLE_ID, goto out);
 	if (peer_version_id < version_id(2, 10, 0))
 		goto unsupported;
+	ERROR_INJECT_YIELD(ERRINJ_NETBOX_ID_DELAY);
```

Comment:
A client can't connect during shutdown, because a server stops accepting new connections before sending out 'shutdown' packets. I need this error injection in order to check if an `IPROTO_ID` request received after the server sent 'shutdown' packets (and hence stopped accepting new connections) is processed correctly.